pycernan/avro/client.py

```diff
@@ -136,6 +38,7 @@ def close(self):
         """
         self.pool.closeall()
 
+    @metrics.publish_failure_count.count_exceptions()
```

Comment:
Where is `count_exceptions` coming from ?

Reply:
It's a convenience method provided by Prometheus client.